PR_GA.md

```diff
@@ -0,0 +1,53 @@
+# General Availability (GA) Requirements for a Pull Request
+
+1. **Code Quality**
+   - The PR must pass all automated tests (unit, integration, end-to-end) without any failures.
+   - The code must follow the project's coding standards and guidelines.
```

Comment:
Do we have any document for these standards ?
Some of them are enforced by linters. Style checks must also pass.

Reply:
I don't think we've got something global, but there is at least the error reporting guidelines on Confluence.
Not sure how to link it here. We might better move such guidelines to the repo.
Style checks are covered by the next item: static code analysis tools.

Reply:
I'd rather not link confluence (internal) documentation from within the code repositories.
I also think if something should be public and referenced from the code repository, it should be in a public code repository.